mii/method_table.py

```diff
@@ -140,19 +142,162 @@ def text2img_pack_response_to_proto(response, time_taken, model_time_taken):
+                                                  time_taken=time_taken,
+                                                  model_time_taken=model_time_taken)
+
+
+class TextClassificationMethods(TaskMethods):
+    @property
+    def method(self):
+        return "ClassificationReply"
+
+    pack_request_to_proto = single_string_request_to_proto
+    unpack_request_from_proto = proto_request_to_single_input
+    pack_response_to_proto = single_string_response_to_proto
+
+
+class QuestionAnsweringMethods(TaskMethods):
+    @property
+    def method(self):
+        return "QuestionAndAnswerReply"
+
+    pack_request_to_proto = question_answering_pack_request_to_proto
+    unpack_request_from_proto = question_answering_unpack_request_from_proto
+    pack_response_to_proto = single_string_response_to_proto
+
+
+class FillMaskMethods(TaskMethods):
+    @property
+    def method(self):
+        return "FillMaskReply"
+
+    pack_request_to_proto = single_string_request_to_proto
+    unpack_request_from_proto = proto_request_to_single_input
+    pack_response_to_proto = single_string_response_to_proto
+
+
+class TokenClassificationMethods(TaskMethods):
+    @property
+    def method(self):
+        return "TokenClassificationReply"
+
+    pack_request_to_proto = single_string_request_to_proto
+    unpack_request_from_proto = proto_request_to_single_input
+    pack_response_to_proto = single_string_response_to_proto
+
+
+class ConversationalMethods(TaskMethods):
+    @property
+    def method(self):
+        return "ConversationalReply"
+
+    pack_request_to_proto = conversational_pack_request_to_proto
+    unpack_request_from_proto = conversational_unpack_request_from_proto
+    pack_response_to_proto = conversational_pack_response_to_proto
+
+
+class Text2ImgMethods(TaskMethods):
+    @property
+    def method(self):
+        return "Txt2ImgReply"
+
+    pack_request_to_proto = multi_string_request_to_proto
+    unpack_request_from_proto = proto_request_to_list
+    pack_response_to_proto = text2img_pack_response_to_proto
+    unpack_response_from_proto = text2img_unpack_response_from_proto
+
+
 GRPC_METHOD_TABLE = {
-    Tasks.TEXT_GENERATION: {
-        "method": "GeneratorReply",
-        "pack_request_to_proto": multi_string_request_to_proto,
-        "unpack_request_from_proto": proto_request_to_list,
-        "pack_response_to_proto": text_generation_pack_response_to_proto,
-        "preprocess_session": text_generation_preprocess_session,
-        "postprocess_session": text_generation_postprocess_session
-    },
+    Tasks.TEXT_GENERATION: TextGenerationMethods(),
+    Tasks.TEXT_CLASSIFICATION: TextClassificationMethods(),
+    Tasks.QUESTION_ANSWERING: QuestionAnsweringMethods(),
+    Tasks.FILL_MASK: FillMaskMethods(),
+    Tasks.TOKEN_CLASSIFICATION: TokenClassificationMethods(),
+    Tasks.CONVERSATIONAL: ConversationalMethods(),
+    Tasks.TEXT2IMG: Text2ImgMethods(),
+}
+"""
```

Comment:
Remove the commented-out old `GRPC_METHOD_TABLE`

Reply:
Done.